src/ui/MenuBar.cpp

```diff
@@ -481,6 +481,19 @@ MenuBar::MenuBar(QWidget *parent)
     dialog->open();
   });
 
+  mRebase = branch->addAction(tr("Squash..."));
+  mRebase->setShortcut(tr("Ctrl+Shift+S"));
```

Comment:
Quick question - Is CTRL+SHIFT+S an universal hotkey shortcut for most git frontend applications for a Squash action?

Reply:
Hi! Sorry for the delay. I searched and found no shortcut key patterns for squash.

Reply:
Very well, I think it's fair enough.

Reply:
Right, no problem.

Reply:
You're resetting the `mRebase` member instead of setting the `mSquash` one you added.